ublox_derive/src/output.rs

```diff
@@ -487,15 +547,31 @@ pub fn generate_code_to_extend_bitflags(bitflags: BitFlagsMacro) -> syn::Result<
         },
     };
 
+    let serde = quote!(#[derive(serde::Deserialize)]);
+
+    let custom_serialize = quote! (
+        impl serde::Serialize for #name {
+            fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+            where
+                S: serde::Serializer,
```

Comment:
```suggestion
    let serde = quote!(#[derive(::serde::Deserialize)]);

    let custom_serialize = quote! (
        impl ::serde::Serialize for #name {
            fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
            where
                S: ::serde::Serializer,
```

Since all this is generated code. Shouldn't they be prefixed with `::`, in case the user for some reason has a module shadowing `serde`?